### PR TITLE
Impl view contract state

### PR DESCRIPTION
**Pull request by silvestrpredko**

### CODE REVIEW

File: tests/rpc.rs [hunk 1]
```diff
@@ -337,6 +338,91 @@ async fn view_access_key_failure() {
...
+        .args(json!({
+            "owner_id": &signer_account_id,
+            "total_supply": "100",
+        }))
+        .gas(near_units::parse_gas!("300 T") as u64)
+        .commit(Finality::Final)
+        .await
+        .unwrap();
+
+    tokio::time::sleep(std::time::Duration::from_secs(2)).await;
```
Comment (kstepanovdev) [inline tests/rpc.rs:368]: Do we need to sleep for that long even in a local chain?

Comment (kstepanovdev) [inline tests/rpc.rs:368]: Do we need to sleep for that long even in a local chain?

File: tests/rpc.rs [hunk 2]
```diff
@@ -337,6 +338,91 @@ async fn view_access_key_failure() {
...
+            DAsNzIsODcuODRWMjAxLjE2QTE1LjM0LDE1LjM0LDAsMCwwLDg3LjM0LDIxNi41aDBhMTUuM\
+            zUsMTUuMzUsMCwwLDAsMTMuMDgtNy4zMWwzMC4xLTQ0LjY5YTMuMiwzLjIsMCwwLDAtNC43N\
+            S00LjJMOTYuMTQsMTg2YTEuMiwxLjIsMCwwLDEtMi0uOTFWMTA0LjYxYTEuMiwxLjIsMCwwL\
+            DEsMi4xMi0uNzdsODkuNTUsMTA3LjIzYTE1LjM1LDE1LjM1LDAsMCwwLDExLjcxLDUuNDNoMy\
+            4xM0ExNS4zNCwxNS4zNCwwLDAsMCwyMTYsMjAxLjE2Vjg3Ljg0QTE1LjM0LDE1LjM0LDAsMCww\
+            LDIwMC42Niw3Mi41aDBBMTUuMzUsMTUuMzUsMCwwLDAsMTg3LjU4LDc5LjgxWicvJTNFJTNDL2c\
+            lM0UlM0Mvc3ZnJTNFAAAY"
+                .to_owned()
+        )
+    );
```
Comment (kstepanovdev) [inline tests/rpc.rs:423]: To be honest, that magic part is just unreadable to me, unfortunately.

Comment (silvestrpredko) [reply tests/rpc.rs:423]: > To be honest, that magic part is just unreadable to me, unfortunately.

It's a contract state represented in a base64 format. A way how to verify it in another way I don't found. Please suggest one if you have one in your mind.

File: tests/rpc.rs (hunk 1 above)
Comment (silvestrpredko) [reply tests/rpc.rs:368]: > Do we need to sleep for that long even in a local chain?

I tested it with 1 second, and sometimes it was not enough.

File: tests/rpc.rs (hunk 2 above)
Comment (kstepanovdev) [reply tests/rpc.rs:423]: Hence we should hope that guys from NEAR won't change their example contract. Neither am I know how to solve this.

**Review by kstepanovdev — APPROVED** [review]

File: tests/rpc.rs (hunk 2 above)
Comment (silvestrpredko) [reply tests/rpc.rs:423]: As a solution, we could have our own contract. Let's do that if this test will fail 😄

**Pull request merged**